### ISSUE TITLE
Disabling Abp settings on tenants

### ISSUE THREAD

**Issue by SeanoNET** [EasyAbp/Abp.SettingUi#40]
Hi!

Thanks for providing this awesome module.

I am seeking advice around disabling the default Abp settings from displaying for tenants. 

For example i would like the host admin to control System and Mail settings and not the individual users or tenant admins.

![image](https://user-images.githubusercontent.com/27134812/94887239-6591c180-04a8-11eb-8bd7-0786844fd423.png)

Is it possible accomplish this in the current version 1.1.10?

**Comment by wakuflair** [comment]
Sorry the feature you want is not implemented currently.
However I think it's pretty simple, there are some discussion: https://github.com/EasyAbp/Abp.SettingUi/issues/12#issuecomment-645169308

Maybe you can PR for it :smile:

**Comment by wakuflair** [comment]
Recently I'm considering adding a "permission" property to a setting property, like:

``` json
  "Test.Setting2": {
    "Group1": "TestGroup1",
    "Group2": "TestGroup2",
    "Type": "checkbox",
    "Permission": "MyProject.MyPermissions.AllowSettingSetting2"
  }
```

In this way, only granted users can see the setting in SettingUI, how about that?

**Comment by SeanoNET** [comment]
@wakuflair Sounds good!

**Comment by cihangll** [comment]
Hi @wakuflair ,
Solution available in this [this](https://github.com/EasyAbp/Abp.SettingUi/pull/46) PR.

**Comment by wakuflair** [comment]
Resolved with #46
